policyengine_us/parameters/gov/states/la/tax/income/exemptions/personal.yaml

```diff
@@ -0,0 +1,22 @@
+description: Louisiana provides an personal exemption based on filing status
+SINGLE:
```

Comment:
need to add widow case

Reply:
I think the documents do not include any information for widow case

Reply:
there should be some reference in the legal code on how widows are treated (usually same as joint)